김진/231206/K번째수.js

```diff
@@ -0,0 +1,20 @@
+function solution(array, commands) {
+  let answer = [];
+  const result = [];
+  for (let i = 0; i < commands.length; i++) {
+    answer = array.slice(commands[i][0] - 1, commands[i][1]).sort();
+    console.log(answer);
+    console.log(
+      answer[commands[i][2]] - 1 == 0
+        ? answer[i][0]
+        : answer[commands[i][2]] - 1
+    );
+    result.push(
+      answer[commands[i][2]] - 1 == 0
```

Comment:
```suggestion
      answer[commands[i][2]] - 1 === 0
```
== 보단 ===를 써주세요.
==는 타입까지 검사하기 때문에 같은 결과를 낸다면 === 사용을 하시면 될 것 같습니다.